apps/s_socket.c

```diff
@@ -239,6 +248,12 @@ int do_server(int *accept_sock, const char *host, const char *port,
     next = BIO_ADDRINFO_next(res);
     if (sock_family == AF_INET6)
         sock_options |= BIO_SOCK_V6_ONLY;
+#ifndef OPENSSL_NO_TFO
+    if (tfo)
+        sock_options |= BIO_SOCK_TFO;
+#else
+    (void)tfo;
```

Comment:
unnecessary?

Reply:
To avoid some compilers complaining of an unused argument when `OPENSSL_NO_TFO` is defined.

Reply:
Yeah - I get why you added it. But `tfo` here is a parameter to the function. We have *many* places in the code where we don't use the argument with no problems - so I'm not sure why this one would be.

Reply:
clang is the usual culprit. But I'm reworking it so it won't matter.